autoload/lsp/symbol.vim

```diff
@@ -101,7 +101,17 @@ def JumpToWorkspaceSymbol(popupID: number, result: number): void
 	exe $'confirm edit {symTbl[result - 1].file}'
       endif
     else
-      winList[0]->win_gotoid()
+      var wid = fname->bufwinid()
+      if wid != -1
+        if bufwinid(bufnr()) == wid
```

Comment:
To check if the current buffer already has the desired file, the following check should be used:
```
      if fname->bufnr() == bufnr()
```